src/app/components/mnoe-api/provisioning.svc.coffee

```diff
@@ -103,4 +103,15 @@ angular.module 'mnoEnterpriseAngular'
           )
       )
 
+    @getSubscriptionEvents = (subscriptionId) ->
+      deferred = $q.defer()
+      MnoeOrganizations.get().then(
+        (response) ->
+          MnoeApiSvc.one('organizations', response.organization.id).one('subscriptions', subscriptionId).one('/subscription_events').get().then(
+            (response) ->
+              deferred.resolve(response)
+          )
+      )
+      return deferred.promise
```

Comment:
I think this can be written like this:
```coffeescript
@getSubscriptionEvents = (subscriptionId) ->
  MnoeOrganizations.get().then(
    (response) ->
      MnoeApiSvc.one('organizations', response.organization.id).one('subscriptions', subscriptionId). customGET('/subscription_events')
  )
```